### PR TITLE
issue-1-see-list-of-food-types | Samuel Cho

### PR DESCRIPTION
**Pull request by Samuel-Cho**
Issue 1: User can see a list of food types
Screen Shot 2021-04-06 at 8 22 43 PM
Screen Shot 2021-04-06 at 8 22 52 PM

### CODE REVIEW

File: server/public/styles.css [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+  margin: 0;
+}
+
+.drawer-container {
+  text-align: center;
+}
+
+.fa-bars {
+  font-size: 1.5rem;
+}
```
Comment (thebearingedge) [inline server/public/styles.css]: Avoid modifying 3rd-party classes. Add your own custom class.

File: server/public/styles.css [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+  font-size: 18px;
+  padding: 10px 30px;
+}
+
+.location-search:hover,
+.food-type:hover,
+.search-button:hover,
+.fa-bars:hover {
+  cursor: pointer;
+}
```
Comment (thebearingedge) [inline server/public/styles.css]: Same here for the custom styling.

File: server/public/styles.css [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+  .app-name {
+    position: absolute;
+    top: 15%;
+    left: 27%;
+    font-size: 2.5rem;
+  }
+
+  .fa-bars {
+    font-size: 2rem;
+  }
```
Comment (thebearingedge) [inline server/public/styles.css]: Here too.

File: server/public/styles.css [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+
+  .search-form {
+    height: 95%;
+
+    /* width: 80%; */
```
Comment (thebearingedge) [inline server/public/styles.css]: Please do not turn in commented out code.

File: server/public/styles.css [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+  }
+
+  .food-type-container {
+    width: 85%;
+    margin-top: 40px;
+  }
+
+  .food-type {
+    /* margin-left: 30px;
+    margin-right: 30px; */
```
Comment (thebearingedge) [inline server/public/styles.css]: 🗑️

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Ok, just a little minor cleanup and then you are good to merge!

**Comment by Samuel-Cho** [comment]
Screen Shot 2021-04-07 at 10 27 45 AM
Screen Shot 2021-04-07 at 10 28 51 AM
Screen Shot 2021-04-07 at 10 28 58 AM
Screen Shot 2021-04-07 at 10 29 14 AM
Screen Shot 2021-04-07 at 10 29 29 AM
Screen Shot 2021-04-07 at 10 29 38 AM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**